server/index.js

```diff
@@ -33,41 +33,72 @@ app.use((req, res, next) => {
   next()
 })
 
-// Remove trailing slashes
+// API Docs - resolve before trailing slash redirect so assets don't break
+
+app.use(
+  '/developers/api',
+  express.static(path.join(__dirname, '../api-docs/build/'), {
+    redirect: false,
+  })
+)
+
+// Remove trailing slashes, UNLESS
+// A) We're in local dev mode, in which case we need the slashes to communicate with the Jekyll server
+// B) Request is for an API docs landing, which needs the slash to not break assets
+// If A or B is true, ensure trailing slash is there
+
+const hasTrailingSlash = reqPath => /.+\/$/.test(reqPath)
+const isApiDocs = reqPath => /^\/developers\/api\/?$/.test(reqPath)
 app.get('*', (req, res, next) => {
-  if (/.+\/$/.test(req.path)) {
+  if (
+    hasTrailingSlash(req.path) &&
+    !(process.env.LOCAL || isApiDocs(req.path))
+  ) {
     res.redirect(req.path.replace(/\/$/, ''))
+  } else if (
+    !hasTrailingSlash(req.path) &&
+    (process.env.LOCAL || isApiDocs(req.path))
+  ) {
+    res.redirect(`${req.path}/`)
   } else {
     next()
   }
 })
 
+
+// Local Development - Proxy requests to local hot-reloading Jekyll server
+
+if (process.env.LOCAL) {
+  const devCenterProxyConfig = {
+    target: config.env.stage.devCenterUrl,
+    changeOrigin: true,
+  }
+  app.use(require('http-proxy-middleware')(devCenterProxyConfig))
```

Comment:
This is actually my fault, as I hadn't yet updated `package.json` when I handed this branch off to you, but I got an error related to `http-proxy-middleware` not being installed when I ran `npm run dev`. Turns out I forgot to add it to package.json so it was never installed.

Reply:
Added!